From 117815c169dbbde56464c39387bc17ee5edd4eaf Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Holger=20Schr=C3=B6der?= <holger.schroeder.ext@basyskom.de>
Date: Thu, 6 Oct 2011 13:49:43 +0200
Subject: [PATCH 5/8] disable qgeocoordinate test

---
diff --git a/tests/auto/qgeocoordinate/qgeocoordinate.pro b/tests/auto/qgeocoordinate/qgeocoordinate.pro
index eb4f47d..2349d20 100644
--- a/tests/auto/qgeocoordinate/qgeocoordinate.pro
+++ b/tests/auto/qgeocoordinate/qgeocoordinate.pro
@@ -18,3 +18,6 @@ symbian {
     INCLUDEPATH += $${EPOCROOT}epoc32/include/osextensions
     TARGET.CAPABILITY = ALL -TCB
 }
+
+maemo*:CONFIG += insignificant_test
+
-- 
1.7.4.1

