From 27f6f1eca334d880dc065b5220feac3ed096cb12 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Holger=20Schr=C3=B6der?= <holger.schroeder.ext@basyskom.de>
Date: Thu, 6 Oct 2011 13:45:32 +0200
Subject: [PATCH 3/8] disable qsystemscreensaver test

---
 .../auto/qsystemscreensaver/qsystemscreensaver.pro |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/tests/auto/qsystemscreensaver/qsystemscreensaver.pro b/tests/auto/qsystemscreensaver/qsystemscreensaver.pro
index 5a98a01..1ed88e1 100644
--- a/tests/auto/qsystemscreensaver/qsystemscreensaver.pro
+++ b/tests/auto/qsystemscreensaver/qsystemscreensaver.pro
@@ -16,3 +16,5 @@ symbian {
 contains(test_use_sim, yes) {
 DEFINES += TESTR QT_SIMULATOR
 }
+
+maemo*:CONFIG += insignificant_test
-- 
1.7.4.1

