diff --git a/tests/auto/qdeclarativecontact/qdeclarativecontact.pro b/tests/auto/qdeclarativecontact/qdeclarativecontact.pro
index ed80c15..b6e1043 100644
--- a/tests/auto/qdeclarativecontact/qdeclarativecontact.pro
+++ b/tests/auto/qdeclarativecontact/qdeclarativecontact.pro
@@ -80,6 +80,9 @@ SOURCES += tst_qdeclarativecontact.cpp \
 
 include(../../../plugins/declarative/common/dynamicproperties.pri)
 
+maemo*:CONFIG += insignificant_test
+
+
 # reserved for future use
 #symbian: {
 #    importFiles.sources = data

