diff --git a/src/harmattaninstalls/rules b/src/harmattaninstalls/rules
index 75c2f13..68308a6 100755
--- a/src/harmattaninstalls/rules
+++ b/src/harmattaninstalls/rules
@@ -24,6 +24,7 @@ DEB_DH_INSTALL_SOURCEDIR := debian/tmp
 # to DEB_DH_INSTALL_SOURCEDIR is not listed in any of the libqtm*.install files
 # except the *.prl files (-X.prl)
 DEB_DH_INSTALL_ARGS := --fail-missing -X.prl
+DEB_DH_MAKESHLIBS_ARGS_ALL := -V
 
 # bearer location contacts multimedia publishsubscribe versit messaging systeminfo serviceframework sensors gallery organizer feedback connectivity
 QTM_MODULES += connectivity

